app/views/state_file/state_file_pages/fake_direct_file_transfer_page.html.erb

```diff
@@ -36,15 +36,15 @@
   <%=
     link_to(
       "Cancel",
-      state_file_questions_canceled_data_transfer_path,
+      StateFile::Questions::CanceledDataTransferController.to_path_helper(us_state: current_intake.state_code),
```

Comment:
I had to change the format of this because these controllers are mapped differently